docs/source/workflows/performance_workflows.rst

```diff
@@ -51,20 +51,20 @@ For each test set, the rest of the data are used to train the predictor, and the
 By comparing the model's predictions to the true values, we can compute several performance metrics that provide information about model quality.
 
 - Root-mean squared error (RMSE): the square root of the average of the squared prediction error
-- Non-dimensional model error (NDME): The RMSE, normalized by the RMSE of a trivial model that always predicts the mean value of the data.
+- Non-dimensional model error (NDME): The RMSE, normalized by the RMSE of a trivial model that always predicts the mean value of the data (== standard deviation of the output variable).
```

Comment:
```suggestion
- Non-dimensional model error (NDME): The RMSE, normalized by the standard deviation of the output variable (which is the RMSE of a trivial model that always predicts the mean value of the data).
```
Your addition made me think that we should just lead with the standard deviation. Feel free to reject if you think this is less legible